### PR TITLE
Login screen

### PR DESCRIPTION
**Pull request by yael-renous**
updated the login screen

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
+        android:label="apartmate"
```
Comment (omerjacobi) [inline app/src/main/AndroidManifest.xml]: why change getting name from string?

File: app/build.gradle [hunk 2]
```diff
@@ -19,6 +20,10 @@ android {
 }
 
 dependencies {
+
+    compile 'com.android.support:appcompat-v7:23.2.0'
```
Comment (omerjacobi) [inline app/build.gradle]: we support from version 27, do we need this line ?

**Review by omerjacobi — CHANGES_REQUESTED** [review]
looks good, just check the comments

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (yael-renous) [reply app/src/main/AndroidManifest.xml]: We changed the app name

File: app/build.gradle (hunk 2 above)
Comment (yael-renous) [reply app/build.gradle]: it can be deleted you're right, I forgot to delete

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (omerjacobi) [reply app/src/main/AndroidManifest.xml]: so change it from /res/string

**Review by omerjacobi — APPROVED** [review]

**Pull request merged**